src/test/unit/requests/publicationRequests.test.ts

```diff
@@ -0,0 +1,62 @@
+import { SummaryOfPublicationsRequests } from '../../../main/resources/requests/summaryOfPublicationsRequests';
+import fs from 'fs';
+import path from 'path';
+import sinon from 'sinon';
+import {dataManagementApi} from '../../../main/resources/requests/utils/axiosConfig';
+
+const rawDataPubs = fs.readFileSync(path.resolve(__dirname, '../../../test/unit/mocks/summaryOfPublications.json'), 'utf-8');
+const pubs = JSON.parse(rawDataPubs);
+const totalCases = 3;
+const pubRequests = new SummaryOfPublicationsRequests();
+const stub = sinon.stub(dataManagementApi, 'get');
+
+beforeEach(async () => {
+  stub.withArgs('/publication/search/0').resolves({data: pubs});
```

Comment:
you only need to do this stub definition once, youre doing it before each test at the mo, should be able to be moved with your stub definition and the before each deleted

Reply:
Should be sorted now :)